src/modules/likeShow.js

```diff
@@ -0,0 +1,10 @@
+const likesShow = (likeID, category) => {
+  const [filteredLikeID] = likeID.filter((idObj) => category.idCategory === idObj.item_id);
```

Comment:
- [ ] Good job for adding counter logic 👌 However, let's ensure to indicate the counter value on how homepage, around the nav section, like seen in the [template design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

**Example of item counter value on homepage nav section:**
![image](https://user-images.githubusercontent.com/12745474/234872386-1dd2f795-4cee-464e-ab00-1ea90bfe572a.png)


**Your nav area:**
![image](https://user-images.githubusercontent.com/12745474/234874219-13562e2d-8a34-45fa-adfa-fd15f16e3089.png)

Reply:
@rloterh done